video_xblock/backends/vimeo.py

```diff
@@ -126,7 +126,14 @@ def advanced_fields(self):
 
         Vimeo videos require Access token to be set.
         """
-        fields_list = super(VimeoPlayer, self).advanced_fields
+        return super(VimeoPlayer, self).advanced_fields
+
+    @property
+    def three_pm_fields(self):
+        """
+        Tuple of VideoXBlock fields to display on `3PlayMedia transcripts` panel.
+        """
+        fields_list = super(VimeoPlayer, self).three_pm_fields
         # Add `token` field before `threeplaymedia_file_id`
         fields_list.insert(fields_list.index('threeplaymedia_file_id'), 'token')
```

Comment:
This doesn't appear to be working correctly.  I only see the field name and help text, but no HTML form fields under token.  See https://github.com/raccoongang/xblock-video/issues/260